settings.gradle

```diff
@@ -44,4 +44,5 @@ include 'entity-registry:custom-test-model'
 include 'metadata-integration:java:spark-lineage'
 include 'metadata-integration:java:datahub-client'
 include 'metadata-integration:java:datahub-protobuf'
-include 'ingestion-scheduler'
\ No newline at end of file
+include 'metadata-ingestion-modules:airflow-plugin'
```

Comment:
is this a new top level module? is there a reason we are pulling out of metadata-ingestion? seems like things could get a bit messy

Reply:
One of the reasons is this module can't share the entry points with other plugins (especially with our other Airflow lineage backend) otherwise you can't separate the two modules. As we have shared entry points as soon as you install the lineage backend, it would load this plugin as well.